### PR TITLE
Fix IgnoreFailedApparentHorizon

### PR DESCRIPTION
**Pull request by geoffrey4444**
## Proposed changes

IgnoreFailedApparentHorizon.hpp includes a nonexistent file, Informer/Tags.hpp. This wasn't caught previously because nowhere in the code actually uses the IgnoreFailedApparentHorizon callback. I updated the KerrSchild executable to ignore AH failures, which are unlikely unless they are deliberate (such as by not including location of the horizon in the domain) for KerrSchild, since the black hole in Kerr Schild doesn't move. This is just to ensure that the code uses IgnoreFailedApparentHorizon somewhere, to catch similar problems in the future.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Review by knelli2 — APPROVED** [review]

**Comment by geoffrey4444** [comment]
macOS failure appears unrelated to this pr

EDIT: this is in fact unrelated, fixed by #3985

**Review by nilsdeppe — APPROVED** [review]
LGTM, long-term we should have a test for the callback, but I don't think that should be part of this PR. We can probably make @markscheel write the test 🙃 

I'll wait for @markscheel to review before merging :)

**Review by markscheel — APPROVED** [review]

**Pull request merged**
